src/main/java/seedu/duke/commands/AddCommand.java

```diff
@@ -16,9 +16,9 @@
     private Task newTask = null;
     private Module newModule = null;
 
-    public AddCommand(String name, String description, boolean isTask) {
+    public AddCommand(String name, String description, boolean isTask, String estimatedWorkingTime) {
         if (isTask) {
-            newTask = new Task(name, description);
+            newTask = new Task(name, description, estimatedWorkingTime);
             isAddTask = true;
         } else {
             newModule = new Module(name, description);
```

Comment:
Suggest to make an overload constructor for task with no estimated working time.